Application/Security.js

```diff
@@ -179,7 +251,7 @@ angular.module('shared')
         currentUser = false;
 
         LoopBackAuth.clearUser();
-        LoopBackAuth.isAdmin = false;
+        //LoopBackAuth.isAdmin = false;
```

Comment:
We used to have a concept of an admin as a user object flag, but this was always flimsy IMO, because the Role/RoleMapping ACL structure is much more powerful and useful, and I think we have successfully standardized on this approach, so this isAdmin concept is deprecated.